File: Code coaches/Medium/Average world length/AverageWordLength.swift
if let words = readLine() {
	let wordsSplitted = words.split(separator: " ")
	
	var wordsNumber = 0.0
	var totalLength = 0.0
	
	for word in wordsSplitted {
		wordsNumber += 1
		for chr in word {
			// Swift 5: if chr.isLetter {
			if ((chr >= "a" && chr <= "z") || (chr >= "A" && chr <= "Z")) {
				totalLength += 1
			}
		}
	}
	
	let averageLength = Int((totalLength / wordsNumber).rounded(.up))
	print(averageLength)
}

File: Code coaches/Medium/Average world length/average_word_length.py
import math

words = input().split(" ")
words_number = 0
total_length = 0

for word in words:
	words_number += 1
	for char in word:
		if char.isalpha():
			total_length += 1

average_length = math.ceil(total_length / words_number)
print(average_length)
